plugins/woocommerce-blocks/assets/js/base/components/select/index.tsx

```diff
@@ -56,6 +62,11 @@ export const Select = ( props: SelectProps ) => {
 							key={ option.value }
 							value={ option.value }
 							data-alternate-values={ `[${ option.label }]` }
+							disabled={
```

Comment:
Added support to have disabled options. I think its simpler and more flexible to allow passing disabled options than to have the select have a concept of a "placeholder" option that would complexify the API of this component (imo) needlessly.

Reply:
Turns out we did need to support this at component level because additional select inputs are also broken